apps/omg_child_chain/lib/omg_child_chain/fee_server.ex

```diff
@@ -49,7 +49,7 @@ defmodule OMG.ChildChain.FeeServer do
       |> Map.put(:expire_fee_timer, nil)
       |> update_fee_specs()
 
-    interval = Map.get(state, :interval_ms, @fee_file_check_interval_ms)
+    interval = Map.get(state, :interval_ms, @fee_adapter_check_interval_ms)
```

Comment:
I don't see this being tested anywhere? 
on the other hand, why bake this into the module (`@fee_adapter_check_interval_ms` and `fee_buffer_duration_ms `). 
this is parametrisation of the process - pass it from the supervision tree down to the process

Reply:
also, side note:
```
    _ = if :undefined == :ets.info(:fees_bucket), do: :ets.new(:fees_bucket, [:set, :public, :named_table])

```
this ets table needs to be private

Reply:
Good point, will do.

Reply:
Actually it cannot. Please look at `FeeServer.current_fees()` it fetches values from `ets` from the caller process (not the FeeServer's GenServer callback)